broker-daemon/orderbook/index.js

```diff
@@ -52,6 +53,21 @@ class Orderbook {
     return getRecords(this.store, MarketEventOrder.fromStorage.bind(MarketEventOrder))
   }
 
+  /**
+   * Get existing block orders
```

Comment:
Change this description to something like `Gets all trades for a specific timestamp`